### DISCUSSION TITLE
Organization with sub brands

### DISCUSSION THREAD

**Discussion by brent-bak** [schemaorg/schemaorg#3051] [Q&A]
If an organization has multiple sub brands, each with its own website, what would be the best way to ensure search engines can understand the relationship between the umbrella organization and the sub brands, and the sub brands to one another?

**Comment by neena90ac** [comment]
Organization --> brand = mention all the brands + url or subOrganisation + other properties
Organization --> subOrganisation +url + other properties

For each sub brand, tag the property parentOrganization.
